src/window.rs

```diff
@@ -34,24 +34,30 @@ use crate::{
 
 use arboard::Clipboard;
 
+/// A panel is a special web view that focus on controlling states around window.
+/// It could be treated as the control panel or navigation bar of the window depending on usages.
+///
+/// At the moment, following Web API is supported:
+/// - Close window: `window.close()`
+/// - Navigate to previous page: `window.prompt('PREV')`
+/// - Navigate to next page: `window.prompt('FORWARD')`
+/// - Refresh the page: `window.prompt('REFRESH')`
+/// - Minimize the window: `window.prompt('MINIMIZE')`
+/// - Maximize the window: `window.prompt('MAXIMIZE')`
+/// - Navigate to a specific URL: `window.prompt('NAVIGATE_TO:${url}')`
+pub struct Panel {
+    pub(crate) webview: WebView,
+    pub(crate) initial_url: servo_url::ServoUrl,
```

Comment:
`initial_url` should only be needed for the panel since it needs to load another webview, the normal webview will have an initial url on create and we don't need to store it

Reply:
I see. Let's move this to `webivew.rs` and document its fields.